components/annonces-section/comptes-association.tsx

```diff
@@ -7,19 +7,24 @@ import { DataSection } from '#components/section/data-section';
 import { FullTable } from '#components/table/full';
 import { EAdministration } from '#models/administrations';
 import { IComptesAssociation } from '#models/annonces';
+import { IAPILoading } from '#models/api-loading';
 import { IAPINotRespondingError } from '#models/api-not-responding';
 import { IAssociation } from '#models/index';
 import { formatDate } from '#utils/helpers';
 
 export const ComptesAssociationSection: React.FC<{
   association: IAssociation;
-  comptesAssociation: IComptesAssociation | IAPINotRespondingError;
+  comptesAssociation:
+    | IComptesAssociation
+    | IAPINotRespondingError
+    | IAPILoading;
 }> = ({ association, comptesAssociation }) => {
   return (
     <DataSection
       data={comptesAssociation}
       title="Dépôt de Comptes des Associations"
       sources={[EAdministration.DILA]}
+      notFoundInfo={null}
```

Comment:
notFoundInfo can be `undefined` why specify `null` here and not just ignore it ?

Reply:
If `null` it means that the block will disappear if no data is found, which was the behavior here before we moved the call to front. 

If not specified (`undefined`), the `Section` is displayed with a generic error message (`AdministrationNotRespondingMessage`).

What's at stakes is to choose if the behaviour « not found » is abnormal (an error is displayed), or if it can happen and just means the association is not concerned (we hide the block).

That being said, I'm not 100% fan of the « hide the block » behaviour, because it looks like a glitch. Something is loading and then suddenly, it's not. It'll be better to let the `Section` block and add a message like « l'association n'a pas déposé de comptes ». 

In short, I'm in favour of removing the `null` behaviour, and force display of a message anyway.